src/sentry/runner/commands/devserver.py

```diff
@@ -52,6 +52,18 @@
         "--synchronize-commit-group=transactions_group",
         "--no-strict-offset-reset",
     ],
+    "post-process-forwarder-generic": [
+        "sentry",
+        "run",
+        "post-process-forwarder",
+        "--entity=search_issues",
+        "--loglevel=debug",
```

Comment:
Hmm, do we need to define the `topic` here?
https://github.com/getsentry/sentry/blob/41b8af967e17f55796569f299794d9e67cf59b12/src/sentry/runner/commands/run.py#L327-L331

Reply:
Should be handled internally if we define `entity` correctly
https://github.com/getsentry/sentry/blob/master/src/sentry/eventstream/kafka/backend.py#L296

Reply:
It is a bit unclear I know... but it gets picked up from https://github.com/getsentry/sentry/blob/a813a868b7bc423785f082758c4fd5eecee96386/src/sentry/eventstream/kafka/backend.py#L295-L296 and the others don't define is so I'd say it's fine to omit